crates/driver/src/main.rs

```diff
@@ -11,19 +12,35 @@ use std::env;
 // #[global_allocator]
 // static ALLOC: Jemalloc = Jemalloc;
 
+#[derive(StructOpt, Debug)]
+#[structopt(name = "driver")]
+struct Opt {
+    /// Verbose mode (-v, -vv, -vvv, etc.)
+    #[structopt(short, long, parse(from_occurrences))]
+    verbose: u8,
```

Comment:
Can you add enum or constant? so that it's clear which level it is, and can be consistent across consumers.

Reply:
I am not sure how we would declare that. Would the previous `enum` value be implied?
Like  `enum Verbosity { AST, ByteCode, EmitResult}`. Or maybe we just add three flags instead of just `-v`. I would be interested to know what other people working on smooth would find convenient. The interpreter is sort of a toy project anyway.

Reply:
that `enum` with `Ord` would work I guess.
or maybe `AST, ASTAndBytecode, ASTAndEmitResult` ?
so that the name implies the order.